tensorflow/python/ops/nn_ops.py

```diff
@@ -519,7 +519,7 @@ def max_pool(value, ksize, strides, padding, data_format="NHWC", name=None):
     strides: A list of ints that has length >= 4.  The stride of the sliding
       window for each dimension of the input tensor.
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
-      See the [comment here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py)
+      See the [comment here](https://www.tensorflow.org/versions/r0.8/api_docs/python/nn.html#convolution)
```

Comment:
@martinwicke do you remember how we can refer to the html documentation without hardcoding the version here?